modules/mobileApp/schemas/pacienteApp.ts

```diff
@@ -110,7 +110,8 @@ PacienteAppSchema.index({
     'pacientes.id': 1
 });
 PacienteAppSchema.index({
-    email: 1
+    email: 1,
+    documento: 1
```

Comment:
Esta bien agregar el indice por documento pero no tiene que ser compuesto.
```suggestion

 PacienteAppSchema.index({
     email: 1
 });
 
 PacienteAppSchema.index({
     documento: 1
 });
```

Son dos indices distintos.

Reply:
Un indice copmpuesto tiene setido si la operación es un ADN 
```
PacienteApp.find({ documento, email });
```

En este caso se busca con un `$or`, mongo en este caso puede utilizar dos indices distintos y después une el resultado.